src/snowflake/cli/plugins/containers/compute_pool/commands.py

```diff
@@ -20,20 +23,42 @@
 def create(
     name: str = typer.Option(..., "--name", help="Name of the compute pool."),
     num_instances: int = typer.Option(
-        ..., "--num", help="Number of compute pool instances."
+        ..., "--num-instances", help="Number of compute pool instances."
     ),
     instance_family: str = typer.Option(
         ...,
         "--family",
         help="Name of the instance family. For more information about instance families, refer to the SQL CREATE COMPUTE POOL command.",
     ),
+    auto_resume: bool = typer.Option(
+        True,
+        "--auto-resume/--no-auto-resume",
+        help="The compute pool will automatically resume when a service or job is submitted to it.",
+    ),
+    initially_suspended: bool = typer.Option(
+        False,
+        "--init-suspend",
+        help="The compute pool will start in a suspended state.",
+    ),
+    auto_suspend_secs: int = typer.Option(
+        3600,
+        "--auto-suspend-secs",
+        help="Number of seconds of inactivity after which you want Snowflake to automatically suspend the compute pool.",
+    ),
+    comment: Optional[str] = comment_option("compute pool"),
     **options,
 ) -> CommandResult:
     """
     Creates a compute pool with a specified number of instances.
     """
     cursor = ComputePoolManager().create(
-        pool_name=name, num_instances=num_instances, instance_family=instance_family
+        pool_name=name,
+        num_instances=num_instances,
```

Comment:
This field name is incorrect. It should be max_nodes and min_nodes for compute pool. max_instances and min_instances are fields for service.